theano/tensor/opt.py

```diff
@@ -1755,7 +1730,7 @@ def local_fill_to_alloc(node):
                        1.52, 'fast_run')
 
 
-@register_canonicalize("fast_compile")
+@register_useless
```

Comment:
I don't have an idea why I would have asked that. Maybe I wasn't clear. Anyway, just add it back.